tools/notification-configuration/notification-creator/NotificationConfigurator.cs

```diff
@@ -186,7 +186,16 @@ private async Task SyncTeamWithCodeOwnerFile(BuildDefinition pipeline, WebApiTea
 
                 // Get contents of CODEOWNERS
                 logger.LogInformation("Fetching CODEOWNERS file");
-                var managementUrl = new Uri(pipeline.Repository.Properties["manageUrl"]);
+                var managementUrl = default(Uri);
+                if (pipeline.Repository.Properties.ContainsKey("manageUrl"))
+                {
+                    managementUrl = new Uri(pipeline.Repository.Properties["manageUrl"]);
+                }
+                if (managementUrl == null)
+                {
+                    logger.LogWarning("There is no url in pipeline. Url: ", pipeline.Repository.Url);
```

Comment:
Do we know what cases don't have a manageUrl? Is that a sign of another issue we need to try and handle?